docs/source/getting-started.rst

```diff
@@ -49,10 +49,18 @@ The easiest way to use Glow in the cloud is with the `Databricks Runtime for Gen
 provider or Spark distribution. You need to install the maven package
 ``io.project:glow_2.11:${version}`` and optionally the Python frontend ``glow.py``.
 
-Example notebook
-----------------
+Notebooks embedded in the docs
+------------------------------
+
+To demonstrate example use cases of Glow functionalities, most of the doc pages on this website are accompanied by embedded notebooks.
+The commands in these notebooks are cloud-independent, except for the ``display()`` command, which is specific to `Databricks Notebooks <https://docs.databricks.com/notebooks/index.html>`_ but can be easily replaced by other commands like ``.show``.
```

Comment:
`df.show` does not work in Python - you'll need the close-paren.

Reply:
removed